Typed Template Haskell/article.md

```diff
@@ -295,6 +302,41 @@ Annotating `mempty'` may resolve it in this case:
 
 An [open ticket](https://gitlab.haskell.org/ghc/ghc/-/issues/10271) exists describing the issue, but if you run into some strange errors, it's a good idea to keep it in mind.
 
+## Compatibility with GHC 9
+
+In GHC 9, the types used by typed Template Haskell were changed according to the [Make Q (TExp a) into a newtype](https://github.com/ghc-proposals/ghc-proposals/blob/master/proposals/0195-code-texp.rst) proposal.
+
+This proposal changed the typed expression quasi-quoter so it now returns `Quote m => Code m a` instead of the old `Quote m => m (TExp a)` from [`template-haskell-2.17.0.0`](https://hackage.haskell.org/package/template-haskell-2.17.0.0/docs/Language-Haskell-TH.html#t:Code).
+
+If the reader is curious, please take a moment to read our [Typed Template Haskell in GHC 9](https://serokell.io/blog/typed-template-haskell-in-ghc-9) article detailing a migration guide of typed TH code to GHC 9, where we discuss a backward-compatible way to integrate typed Template Haskell code in both GHC 8 and GHC 9.
```

Comment:
```suggestion
If the reader is curious, please take a moment to read our [Typed Template Haskell in GHC 9](./typed-template-haskell-in-ghc-9) article detailing a migration guide of typed TH code to GHC 9, where we discuss a backward-compatible way of integrating typed Template Haskell code in both GHC 8 and GHC 9.
```

I am not sure about absolute vs. relative URLs, what do we normally use? @NaeosPsy? I mean, I am personally a fan of relative URLs, I think we should use them whenever possible, but maybe there are reasons to prefer absolute?

Reply:
We have historically used absolute, and I think it would be simpler to leave it at that.

Reply:
Ok, but if we rename the company and change the domain, you’ll have to go fix it all 🙄